users/static/user_site/home.js

```diff
@@ -1,23 +1,81 @@
-const toggle = document.querySelector('.toggle');
-const calender = document.querySelector('.calender');
-
-toggle.addEventListener('click', (e)=> {
-	toggleClass(calender, 'cal-view');
-});
-
-let toggleClass = (el, className) => {
-	if (el.classList) {
-		el.classList.toggle(className);
-	} else {
-		let classes = el.className.split(' ');
-		const existingIndex = classes.indexOf(className);
-
-		if (existingIndex >= 0) {
-			classes.splice(existingIndex, 1);
-		} else {
-			classes.push(className);
-		}
-
-		el.className = classes.join(' ');
-	}
-};
+
+function set_line() {
+	const d = new Date();
+	document.querySelector(".dayview-now-marker").style.top =
+	(document
+		.querySelector(".dayview-gridcell-container")
+		.getBoundingClientRect().height / 24) *
+		(d.getHours() + d.getMinutes() / 60) + "px";
+}
```

Comment:
The red line represents the current time appearing in every day-view, and I think it should appear only on the current date.

![image](https://user-images.githubusercontent.com/63968945/167883065-84ca59af-58c3-490e-a2e9-3a72935b7414.png)

Reply:
done. 
Great observation.
Thanks 😄